themes/nodebb-theme-persona/templates/partials/groups/userOnlineCount.tpl

```diff
@@ -0,0 +1 @@
+<h6> ({groups.onlineUserCount}10) </h6>
```

Comment:
The 10 should be removed. 
Not a big deal. Can be removed in another PR if necessary.

Reply:
I could remove the 10 now it's just a placeholder so something shows up on the UI.